src/tools/r2rdump/UnwindInfo.cs

```diff
@@ -6,52 +6,46 @@
 
 namespace R2RDump
 {
```

Comment:
The contents of this file are architecture specific, but the file/functionality isn't named in an architecture neutral way.